example/full/instance.properties

```diff
@@ -1350,3 +1350,16 @@ sleeper.default.statestore.committer.update.every.commit=false
 # When using the transaction log state store, this sets whether to update from the transaction log
 # before adding a batch of transactions in the asynchronous state store committer.
 sleeper.default.statestore.committer.update.every.batch=true
+
+# Default value for the concurrency assigned to the lambdas across the entire of the sleeper instance.
+# Each lambda also has its own implementation of the property which will override the value found
+# here.
+# See reserved concurrency overview at:
+# https://docs.aws.amazon.com/lambda/latest/dg/configuration-concurrency.html
+# sleeper.default.lambda.concurrency.reserved=
+
+# Default value for the maximum concurrency assigned to the lambdas across the entire of the sleeper
```

Comment:
Similar to above, this might be better as "Default value for the maximum concurrency assigned to each lambda within the Sleeper instance."

Reply:
Addressed